### ISSUE TITLE
Method "setImage" has a problem

### ISSUE THREAD

**Issue by CyrusHeo** [lsnepomuceno/laravel-a1-pdf-sign#72]
Param $page doesn't work . 
 
**
check the file here : lsnepomuceno/laravel-a1-pdf-sign/src/SignaturePdf.php Line: 266

When $this->hasSealImgOnEveryPages === false , The correct codes should be like this :

```
if ($this->hasSealImgOnEveryPages) {
                $this->implementSignatureImage($i);
            }else{
                extract($this->image);
                if($page == $i){
                    $this->implementSignatureImage($page);
                }
            }
```

**Comment by lsnepomuceno** [comment]
As the issue template was not used, informing important versions to identify the problem, I will consider the problem as a "false positive".

If the issue is sent again, please use the consistent template containing the information necessary to identify the problem.

**Comment by lsnepomuceno** [comment]
Hello @CyrusHeo ,

The fix was implemented in PR https://github.com/lsnepomuceno/laravel-a1-pdf-sign/pull/85 and is now available from version [v1.0.7](https://github.com/lsnepomuceno/laravel-a1-pdf-sign/releases/tag/1.0.7)
